test/test_explicit_bool_parameter.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import argparse
+from unittest.mock import patch
+
+import luigi
+import luigi.mock
+from luigi.cmdline_parser import CmdlineParser
+
+import gokart
+
+
+def run_locally(args):
+    temp = CmdlineParser._instance
+    try:
+        CmdlineParser._instance = None
+        run_exit_status = luigi.run(['--local-scheduler', '--no-lock'] + args)
+    finally:
+        CmdlineParser._instance = temp
+    return run_exit_status
+
+
+class WithDefaultTrue(gokart.TaskOnKart):
+    param = gokart.parameter.ExplicitBoolParameter(default=True)
+
+
+class WithDefaultFalse(gokart.TaskOnKart):
+    param = gokart.parameter.ExplicitBoolParameter(default=False)
+
+
+class ExplicitParsing(gokart.TaskOnKart):
+    param = gokart.parameter.ExplicitBoolParameter()
+
+    def run(self):
+        ExplicitParsing._param = self.param
+
+
+class TestExplicitBoolParameter(unittest.TestCase):
+    def test_bool_default(self):
+        self.assertTrue(WithDefaultTrue().param)
+        self.assertFalse(WithDefaultFalse().param)
+
+    def test_parse_param(self):
+        run_locally(['ExplicitParsing', '--param', 'true'])
```

Comment:
[IMO] As like this way (https://github.com/spotify/luigi/blob/master/test/dict_parameter_test.py#L49), Parser can be tested without  run. (Or unittest should be run without runnning CLI)

Reply:
Thx for your comments!
I rewrite code to use `in_parse` instead of `run_loccaly`.

Reply:
🆒